bus-mapping/src/circuit_input_builder/call.rs

```diff
@@ -91,10 +89,14 @@ pub struct Call {
     pub return_data_offset: u64,
     /// Return data length
     pub return_data_length: u64,
+    /// Last Callee's id.
+    pub last_callee_id: usize,
     /// last callee's return data offset
     pub last_callee_return_data_offset: u64,
     /// last callee's return data length
     pub last_callee_return_data_length: u64,
+    /// last callee's memory
+    pub last_callee_memory: Memory,
```

Comment:
the reason why last_callee_memory is needed, is that we cannot read masked bytes from call.return_data if addr is not aligned?

Reply:
Correct ! need to get entire memory not only returndata.